sqlite3/lib/src/common/database.dart

```diff
@@ -173,6 +173,9 @@ abstract class CommonDatabase {
     bool directOnly = true,
   });
 
+  /// Waits for the data to be written to persistent storage
+  Future<void> flush();
+
```

Comment:
This API definitely shouldn't be on the common database since it doesn't make sense for all implementations. If at all, we should only add it to the wasm database.